project_setup/scripts/terraform-to-secrets

```diff
@@ -130,6 +130,8 @@ def parse_tagged_resources(
     """Search all resources for tags requesting the creation of a secret."""
     for resource in find_resources(terraform_state, None):
         tags: Dict[str, str] = resource["values"].get("tags", dict())
```

Comment:
We might need a comment here to our future selves.  I know that if I ever look at this I'll assume this is redundant code.  Get `get("tags", dict())` is guarding against unset keys in `resource["values"]` while the new `if` clause is guarding against `tags` being explicitly set to `None`. 

It might be clearer to remove the `default` argument to `get` and just let the `if` block handle everything.  (Still with a comment, cause I know I'll later say, "Gee, they should have used the `default` arg")

Reply:
Excellent suggestion @felddy.  Please take a look at https://github.com/cisagov/development-guide/pull/11/commits/99545185ac48c295d3217c7c6692f45b9f057aa3 and let me know if that works for you.